projects/storefrontlib/src/cms-components/product/product-tabs/product-attributes/product-attributes.component.html

```diff
@@ -11,6 +11,16 @@ <h3>{{ class.name }}</h3>
           <ul>
             <li *ngFor="let featureValue of feature?.featureValues">
               {{ featureValue?.value }}
+
+              <span
+                *ngIf="
+                  feature.featureUnit.symbol &&
+                  feature.featureUnit.symbol.length > 0 &&
+                  feature.featureUnit.symbol !== '.'
+                "
```

Comment:
Could be simplified with 
```
*ngIf="
  feature.featureUnit?.symbol?.length > 0 &&
   feature.featureUnit.symbol !== '.'
"
```

Perhaps you could also filter on unit type "300" rather than symbol "." for a the no units values.

Reply:
Made the fix.